chainercv/links/connection/convolution_2d_block.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+
+import chainer
+from chainer.functions import relu
+from chainer.links import BatchNormalization
+from chainer.links import Convolution2D
+
+
+class Convolution2DBlock(chainer.Chain):
+    """Convolution2D --> (Batch Normalization) --> Activation
```

Comment:
It is better to give more specific name because some networks use `BN->ReLU->Conv` block.